library/ssl_tls13_client.c

```diff
@@ -1091,6 +1278,14 @@ static int ssl_tls13_parse_server_hello( mbedtls_ssl_context *ssl,
         p += extension_data_len;
     }
 
+    if( !supported_versions_ext_found )
```

Comment:
Remove supported_version_ext_found . And replace it with `handshake->extensions_present & EXT_SUPPORTED_VERSION`